frontend/src/app/tasks/components/EditTask.tsx

```diff
@@ -0,0 +1,115 @@
+import Box from '@mui/joy/Box'
+import Stack from '@mui/joy/Stack'
+
+import { Task } from '@/domain/Task'
+import { TaskType } from '@/domain/TaskType'
+import { Project } from '@/domain/Project'
+import Button from '@mui/joy/Button'
```

Comment:
This is very minor, but can we move the button nearer the other joy components? just to keep the imports more organized